project/AppDependencies.scala

```diff
@@ -17,7 +17,7 @@ object AppDependencies {
     "uk.gov.hmrc"         %% "tax-year"                 % "1.1.0",
     "uk.gov.hmrc"         %% "time"                     % "3.19.0",
     "uk.gov.hmrc"         %% "domain"                   % s"8.0.0-$playVersion",
-    "uk.gov.hmrc.mongo"   %% s"hmrc-mongo-$playVersion" % "0.62.0",
+    "uk.gov.hmrc.mongo"   %% s"hmrc-mongo-$playVersion" % "0.67.0",
```

Comment:
Can you just have the test dependency as 0.67.0? The bobby rule will trigger for anything below 0.68.0 so this will still fail won't it?